src/proto/grpc/binary_log/v1alpha/log.proto

```diff
@@ -105,4 +105,4 @@ message Message {
   // The contents of the message. May be a prefix instead of the complete
   // message.
   bytes data = 5;
-}
```

Comment:
I'm skeptical. If GitHub is showing us a diff, `git` must be smart enough to detect the diff.

Please remove the file from the pull request unless a "real" change needs to be made?